src/promptflow/promptflow/_cli/_pf/_flow.py

```diff
@@ -298,7 +298,12 @@ def _init_flow_by_template(flow_name, flow_type, overwrite=False):
     copy_extra_files(flow_path=flow_path, extra_files=["requirements.txt", ".gitignore"])
 
     print(f"Done. Created {flow_type} flow folder: {flow_path.resolve()}.")
-    flow_test_args = "--interactive" if flow_type == "chat" else f"--input {os.path.join(flow_name, 'data.jsonl')}"
+    if flow_type == "chat":
+        flow_test_args = "--interactive"
+        print("The generated chat flow is requiring a connection named open_ai_connection, "
+              "please follow the steps in README.md to create if you haven't done that.")
```

Comment:
How do we make sure user can find the `README.md`?

Reply:
![image](https://github.com/microsoft/promptflow/assets/17938940/936fb943-fa9b-4b66-b3a8-736528463db4)
README.md is in chat template

Reply:
I know where it is, i meant how could customer know this when seeing this message while he's just running commands in a random terminal